daisy_workflows/image_build/enterprise_linux/export_metadata.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Export image metadata on a GCE VM.
+
+Parameters (retrieved from instance metadata):
+google_cloud_repo: The package repo to use. Can be stable (default), staging,
+  or unstable.
+el_release: rhel6, rhel7, centos6, centos7, oraclelinux6, or oraclelinux7
+image_dest: The Cloud Storage destination for the resultant image.
+destination_prefix: The CS path to export image metadata to
+"""
+
+from datetime import datetime, timezone
+import json
+import logging
+import os
+
+import utils
+
+
+def main():
+  # Get Parameters.
+  repo = utils.GetMetadataAttribute('google_cloud_repo',
+                                    raise_on_not_found=True)
```

Comment:
Interesting -- so the new code for Debian and FAI lives in an existing `build` file, while for EL it gets its own new file. I'm left wondering why the same pattern wasn't used for all three (either combine into existing build files, or create entirely new files)

Reply:
While liam has explain this to me before. The debian and el workflow is not follow the same pattern. Make it simple, the el workflow separate the export image step to using export tool.